README.md

```diff
@@ -14,4 +14,4 @@ The goal of this project is to show one way to to set up split in Angular.
 
```

Comment:
we could add some comment like the one [here](https://github.com/splitio/react-sdk-examples#running-the-example), to explain that users must update the Api key and check that evaluated splits exist in their workspace.